.buildkite/pipeline.yml

```diff
@@ -35,23 +35,19 @@ steps:
         run: unit-test
     command: 'bundle exec rake'
 
-  - label: 'Comparison tests'
+  - label: 'No-device tests'
```

Comment:
I've refactored this into a single steps to avoid asking for several AWS instances, for a brief period of time (which will then take time to scale down).